articles/active-directory/develop/scenario-daemon-overview.md

```diff
@@ -34,6 +34,11 @@ Here are some examples of use cases for daemon apps:
 
 There's another common case where non-daemon applications use client credentials: even when they act on behalf of users, they need to access a web API or a resource under their own identity for technical reasons. An example is access to secrets in Azure Key Vault or Azure SQL Database for a cache.
 
+Note, **the daemon applications cannot be deployed to regular users’ devices or be accessed by regular users**, only limited set of IT administrator can access devices which have daemon apps running, otherwise a bad actor can access client secret or tokens from the traffic of the device and behave on behalf of the daemon application. The daemon app scenario is not replacement of device authentication.
```

Comment:
Can you add a link to Microsoft docs article explaining device authentication?

Reply:
I'm not sure we have it public.

Reply:
Can you create a work item to create documentation? Otherwise for developers like my team this is a dead end since we won't know how to proceed to build the capability we need.